composer.json

```diff
@@ -14,7 +14,8 @@
         }
     },
     "require": {
-        "amphp/artax": "^3",
+        "php": "^7.0",
```

Comment:
That said, PHP 8.0 is _probably_ far enough away, so it's not too much work to bump it once every 5-10 years.